src/main/rp/RP.ts

```diff
@@ -98,9 +123,22 @@ export class RP {
     const verifyAuthenticationResponseOpts = this.newVerifyAuthorizationResponseOpts({
       ...opts,
     });
-    const authorizationResponse = await AuthorizationResponse.fromPayload(authorizationResponsePayload);
-
-    return await authorizationResponse.verify(verifyAuthenticationResponseOpts);
+    const authorizationResponse = await AuthorizationResponse.fromPayload(authorizationResponsePayload).catch((error: Error) => {
```

Comment:
then, catch instead of catch and emit outside of the flow. So similar to the approach done below

Reply:
cannpt really do that here as we need authorizationResponse later below